Frontend/ui-customer/src/Screens/CustomerUtils/SavedLocationScreen/View.purs

```diff
@@ -71,7 +76,38 @@ screen initialState st =
   }
 
 view :: forall w. (Action -> Effect Unit) -> ST.SavedLocationScreenState -> PrestoDOM (Effect Unit) w
-view push state =
+view push state = do
+  Anim.screenAnimation $ relativeLayout
+    [ height MATCH_PARENT
+    , width MATCH_PARENT
+    , background Color.white900
+    , padding $ Padding 0 EHC.safeMarginTop 0 EHC.safeMarginBottom
+    ][ GenericHeader.view (push <<< GenericHeaderAC) (genericHeaderConfig state)
+    , linearLayout[
+        height $ V 1
+      , width MATCH_PARENT
+      , background Color.grey800
+      , margin $ MarginTop 60
+    ][]
+    , tabView push state
+    , if state.data.current == "Drivers" then driverView push state else locationView push state
```

Comment:
@ROHIT-43  state.data.current == "Drivers" 
Can we have instead of string check?

Reply:
should i use enum instead of String